flujo/cli/config.py

```
@@ -238,16 +238,30 @@ def load_backend_from_config() -> StateBackend:
         if not parent_dir.exists():
-            typer.echo(
-                f"[red]Error: Database directory '{parent_dir}' does not exist[/red]",
-                err=True,
-            )
+            try:
+                from rich.console import Console as _Console
+
+                _Console(stderr=True).print(
+                    f"[red]Error: Database directory '{parent_dir}' does not exist[/red]"
+                )
+            except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)